mysql-test/std_data/dd/sdi/innodb_sdi/mysql.json

```diff
@@ -24735,7 +24735,7 @@
         "created": NNN,
         "last_altered": NNN,
         "hidden": 1,
-        "options": "avg_row_length=0;explicit_tablespace=1;key_block_size=0;keys_disabled=0;pack_record=0;stats_auto_recalc=0;stats_persistent=0;stats_sample_pages=0;",
+        "options": "avg_row_length=0;encrypt_type=N;explicit_tablespace=1;key_block_size=0;keys_disabled=0;pack_record=0;stats_auto_recalc=0;stats_persistent=0;stats_sample_pages=0;was_encrypt_type_set_explicitly=1;",
```

Comment:
In this diff there is no mysql.host. Other than that actually those tables are created with explicit encryption type. Have a look at mysql_system_tables.sql. It would make sense to remove the encryption clause from there so the ENCRYPTION clause was not added in case table is not encrypted. Otherwise encryption threads will not pick up those tables. I do not know if it is worth the effort since we will probably go with NO_ONLINE. Anywas, I will change it.